### PR TITLE
Styles for MainActivity

### PR DESCRIPTION
**Pull request by jsburgin**
Going to push more to this in the morning before merging, just wanted to get it up tonight. Still need to finish up styling the favorites and recommendation buttons.

### CODE REVIEW

File: app/src/main/java/net/jmhossler/roastd/maintask/MainActivity.java [hunk 1]
```diff
@@ -21,6 +24,8 @@ protected void onCreate(Bundle savedInstanceState) {
         mainFragment, R.id.mainFrame);
     }
 
-    mMainPresenter = new MainPresenter(mainFragment);
+    FirebaseAuth firebaseAuth = FirebaseAuth.getInstance();
+
+    mMainPresenter = new MainPresenter(mainFragment, firebaseAuth);
```
[the remark this answers is not in the document]
Comment (jsburgin) [reply app/src/main/java/net/jmhossler/roastd/maintask/MainActivity.java]: Already on it 😄

File: app/src/main/res/drawable/ic_search.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<vector android:height="16dp" android:viewportHeight="512"
+    android:viewportWidth="512" android:width="16dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#FFEA5C41" android:pathData="M508.5,468.9L387.1,347.5c-2.3,-2.3 -5.3,-3.5 -8.5,-3.5h-13.2c31.5,-36.5 50.6,-84 50.6,-136C416,93.1 322.9,0 208,0S0,93.1 0,208s93.1,208 208,208c52,0 99.5,-19.1 136,-50.6v13.2c0,3.2 1.3,6.2 3.5,8.5l121.4,121.4c4.7,4.7 12.3,4.7 17,0l22.6,-22.6c4.7,-4.7 4.7,-12.3 0,-17zM208,368c-88.4,0 -160,-71.6 -160,-160S119.6,48 208,48s160,71.6 160,160 -71.6,160 -160,160z"/>
```
Comment (jmhossler) [inline app/src/main/res/drawable/ic_search.xml]: I applaud your dedication to making things look good. Thanks for doing this. It looks like gibberish to me.

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (jmhossler) [inline app/src/main/res/layout/activity_main.xml]: any particular reason for going with relative layout over linear layout? I'm personally not too sure on the differences, only reason I ask.

File: app/src/main/java/net/jmhossler/roastd/maintask/MainActivity.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (jmhossler) [reply app/src/main/java/net/jmhossler/roastd/maintask/MainActivity.java]: rad

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (jsburgin) [reply app/src/main/res/layout/activity_main.xml]: Everything is relative to sibling elements. The search button is centered in the view and all the labels above it depend on it.

https://developer.android.com/guide/topics/ui/layout/relative.html

**Comment by jmhossler** [comment]
looks cool so far; excited to see the style you apply to the favorites button in the morning. Good job!

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (jmhossler) [reply app/src/main/res/layout/activity_main.xml]: okay, cool. This clarifies things for me.

**Comment by jmhossler** [comment]
holding off approval for changes in the morning.

File: app/src/main/java/net/jmhossler/roastd/maintask/MainFragment.java [hunk 4]
```diff
@@ -89,8 +90,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     mProfile = root.findViewById(R.id.profile);
     mProfile.setOnClickListener(v -> startActivityForResult(new Intent(getContext(), ProfileActivity.class), RC_SIGN_OUT));
 
-    mFavoritesButton = root.findViewById(R.id.start_favorites_activity_button);
-    mFavoritesButton.setOnClickListener(v -> startActivity(new Intent(getContext(), FavoritesActivity.class)));
+    //mFavoritesButton = root.findViewById(R.id.start_favorites_activity_button);
```
Comment (jmhossler) [inline app/src/main/java/net/jmhossler/roastd/maintask/MainFragment.java]: why are we removing the favorites button?

Comment (jsburgin) [reply app/src/main/java/net/jmhossler/roastd/maintask/MainFragment.java]: It's still there was just pushing where I merged in search and it was commented out atm.

**Pull request merged**